### ISSUE TITLE
Can´t join Whatsapp groups

### ISSUE THREAD

**Issue by pasith3e** [42wim/matterbridge#1801]
**Describe the bug**
Can´t use Whatsapp Groups after update to v1.25.0/whatsmeow. 
Since updating to the latest Version Matterbridge crashes at startup when a Whatsapp Gateway configured.

**To Reproduce**
1. Configure Matterbridge with a Whatsapp Group as gateway
2. Start Matterbridge 

**Expected behavior**
When starting Matterbridge with a gateway.out or gateway.inout configured that is linked to Whatsapp account Matterbridge crashes at startup with the Message: 
[0000] FATAL main:         [main:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/matterbridge.go:66] Starting gateway failed: Bridge whatsapp.bridge failed to join channel: please specify group's JID from the list above instead of the name 'xxxxxx-xxxxxxx@g.us'
There is no output of an JID in the output.
I tried it with the Whatpsapp Groupname, Empty Strings, combinations of the Phone number and GroupID as channel but nothing works.

**Screenshots/debug logs**
```
matterbridge -debug -conf matterbridge.toml 
[0000]  INFO main:         [setupLogger:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/matterbridge.go:104] Enabling debug logging.
[0000]  INFO main:         [main:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/matterbridge.go:44] Running version 1.25.0 
[0000]  INFO router:       [Start:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/gateway/router.go:66] Parsing gateway gateway 1
[0000]  INFO router:       [Start:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/gateway/router.go:75] Starting bridge: whatsapp.bridge 
[0000] DEBUG whatsapp:     [Connect:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/bridge/whatsappmulti/whatsapp.go:74] Connecting to WhatsApp..
[0000]  INFO whatsapp:     [Connect:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/bridge/whatsappmulti/whatsapp.go:116] WhatsApp connection successful
[0000]  INFO whatsapp:     [Connect:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/bridge/whatsappmulti/whatsapp.go:134] Getting user avatars..
[0000]  INFO whatsapp:     [Connect:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/bridge/whatsappmulti/whatsapp.go:149] Finished getting avatars..
[0000]  INFO whatsapp:     [joinChannels:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/bridge/bridge.go:77] whatsapp.bridge: joining 49xxxxxxxx-xxxxxxxx@g.us (ID: 49xxxxxxx-xxxxxxx@g.uswhatsapp.bridge)
18:42:27.820 [Client INFO] Successfully authenticated
18:42:28.087 [Client INFO] Uploading 50 new prekeys to server
[0000]  INFO whatsapp:     [JoinChannel:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/bridge/whatsappmulti/whatsapp.go:199] xxxxxx@g.us TestGruppe
[0000] FATAL main:         [main:/usr/local/go/pkg/mod/github.com/42wim/matterbridge@v1.25.0/matterbridge.go:66] Starting gateway failed: Bridge whatsapp.bridge failed to join channel: please specify group's JID from the list above instead of the name 'xxxxxx-xxxxxxxx@g.us'

```

**Environment (please complete the following information):**
 - OS:  Linux
 - Matterbridge version: version: 1.25.0
 - Installed via go install -tags nomsteams,whatsappmulti github.com/42wim/matterbridge@master same error when using [Dockerfile_whatsappmulti](https://github.com/42wim/matterbridge/blob/master/Dockerfile_whatsappmulti)

**Additional context**
```

Token="xxxxxxxxxxx"
RemoteNickFormat="({PROTOCOL}) &lt;{NICK}&gt; "
MessageFormat="Markdown"

[whatsapp.bridge]
#[whatsapp.mywhatsapp]
# Number you will use as a relay bot. Tip: Get some disposable sim card, don't rely on 
# your own number.
Number="49xxxxxx"

# First time that you login you will need to scan QR code, then credentials will be saved in 
# a session file. 
# If you won't set SessionFile then you will need to scan QR code on every restart.
# optional (by default the session is stored only in memory, till restarting matterbridge)
SessionFile="session-xxxxxx.gob"


# If your terminal is white we need to invert QR code in order for it to be scanned properly
# optional (default false)
QrOnWhiteTerminal=false

# Messages will be seen by other WhatsApp contacts as coming from the bridge. 
# Original nick will be part of the message.
RemoteNickFormat=""

[[gateway]]
name="gateway 1"
enable=true

#[[gateway.inout]]
[[gateway.out]]
account="whatsapp.bridge"
channel="xxxxxxx-xxxxx@g.us"


[[gateway.inout]]
account="telegram.mytelegram"
channel="xxxxxxxxx"

```

Thanks for this amazing Project.
And thanks in advance for looking into this. 
Any Help is appreciated

**Comment by 42wim** [comment]
Are you using whatsapp beta?

**Comment by pasith3e** [comment]
Hi,

No I´m no using the Whatsapp Beta but I found the Problem.

1. I was trying to join a Broadcast Group/Channel which is not supported by the new Whats app backend.
2. The JID totally changed and can´t be derived from the group Invite URL anymore. I found a working one via the listgroups call in the mdtest tool in the https://github.com/tulir/whatsmeow repo.

To avoid further problem maybe dump to the output of GetJoinedGroups() when an empty or no channel is provided. 
Thank you for you help.
